pkg/operator/azurerm/resourcestatus/converter.go

```diff
@@ -0,0 +1,90 @@
+package resourcestatus
+
+import "github.com/seal-io/walrus/pkg/dao/types/status"
+
+// virtualMachineStatusConverter generate the summary use following table, other status is treated as transitioning.
+//
+// | Human Readable Status | Human Sensible Status |
+// | --------------------- | --------------------- |
+// | running               |                       |
+// | stopped               |                       |
```

Comment:
Need to change the function NewConverter to support convert stopped status to inactive